app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Api::V1::TasksController < Api::V1::ApplicationController
+  def index
+    q_params = params[:q] || { s: 'id asc' }
+
+    tasks = Task.ransack(q_params)
+                .result
+                .page(params[:page])
+                .per(params[:per_page])
+
+    json = {
+      items: tasks.map { |t| TaskSerializer.new(t).as_json },
+      meta: build_meta_tasks(tasks)
+    }
+
+    respond_with json
+  end
+
+  def show
+    task = Task.find(params[:id])
+    respond_with(task)
+  end
+
+  def create
+    task = current_user.my_tasks.new(task_params)
+
+    if task.save
+      respond_with(task, location: nil)
+    else
+      respond_with(task.errors, status: :unprocessable_entity)
```

Comment:
What do you think will happen if you just write
```
task.save
respond_with task
```

Reply:
thanks!
https://www.rubydoc.info/github/plataformatec/responders/ActionController%2FRespondWith:respond_with
http://blog.plataformatec.com.br/2009/08/embracing-rest-with-mind-body-and-soul/
It helped to understand. I hope now it is correct.